content/software-security/compliance/cmmc-2/cmmc-practices.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Overview of CMMC 2.0 Practices/Control Groups "
+description: "Overview of CMMC 2.0 Practices/Control Groups "
+lead: "Learn about the 14 differenct domains of practices required for CMMC 2.0."
+type: "article"
+date: 2024-08-09T19:10:09+00:00
+lastmod: 2024-08-09T19:10:09+00:00
+contributors: ["Erin Glass"]
+draft: false
+tags: ["compliance", "CMMC 2.0", "standards"]
+images: []
+menu:
+  docs:
+    parent: "cmmc-2"
+weight: 003
+toc: true
+---
+
+In total, Cybersecurity Maturity Model Certification (CMMC) 2.0 has 130 practices, though only Level 3 needs to meet them all. These practices are divided into 14 domains, each of which covers a different aspect of cybersecurity.
+
+Wait, you may be wondering. Are “practices” the same as “controls” or “requirements?”  This can be a point of confusion, as CMMC 2.0 refers to its requirements as “practices”, yet the majority are taken from NIST SP 800-171 and NIST SP 800-172 standards or requirements.
+
+## Naming Conventions for CMMC 2.0 Practices/Controls
+
+CMMC 2.0 practices are labeled using the format “DD.L#-REQ”, where:
+- DD is the two-letter domain abbreviation
+- L# is the level number
+- REQ is the NIST SP 800-171 Rev 2 or NIST SP 800-172 security requirement number
+
+## CMMC 2.0 Practice/Control Domains
+
+Below is a table overview of the domains:
+
+| Domain                          | Description                                                                                                   | Example Practice/Control                                     |
+|---------------------------------|---------------------------------------------------------------------------------------------------------------|-----------------------------------------------------|
+| **Access Control (AC)**          | Manages and restricts access to information systems to ensure that only authorized users and processes can interact with the system, protecting sensitive data from unauthorized access. | AC.L1-3.1.22 - Control information posted or processed on publicly accessible information systems. |
```

Comment:
It threw me for a moment when the verbs changed from third person singular (manages and restricts) to imperative (control information). I see the parallel across all the domain descriptions/examples and see what's going on, just noting. I'm not sure there's anything to be done here.

Reply:
You're right! I didn't notice because I did each column separately. Changing both to imperative. Thanks!